Sources/StreamChat/Query/Sorting/ChannelMemberListSortingKey.swift

```diff
@@ -32,7 +32,7 @@ extension ChannelMemberListSortingKey {
         return .init(keyPath: dateKeyPath, ascending: false)
     }()
     
-    func sortDescriptor(isAscending: Bool) -> NSSortDescriptor? {
+    func sortDescriptor(isAscending: Bool) -> NSSortDescriptor {
```

Comment:
This was not a fix, but having this optional was not required, and it wasn't public API